packaging/packaging.go

```diff
@@ -49,6 +50,8 @@ type FilePackager struct {
 	uid      string
 	MaxSize  int64
 	maxBytes int64
+	Start    time.Time
+	End      time.Time
```

Comment:
These fields do not need to be exported, so let's lowercase them.

Reply:
done!